src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.starknet.common.syscalls import get_caller_address, replace_class, library_call
+
+const INITIALIZE_SELECTOR = 0x79dc0da7c54b95f10aa182ad0a46400db63156920adb65eca2654c0945a463;  // sn_keccak('initialize')
+
+@storage_var
+func Account_evm_address() -> (evm_address: felt) {
+}
+
+@storage_var
+func Account_kakarot_address() -> (kakarot_address: felt) {
+}
```

Comment:
maybe add a comment here to say that we use on purpose storage_slot collision (meaning that these storage var are also defined in the library, but we don't want to import it, so we redefine them. Add also a comment in the library next to them so that if at some point we change the name, we don't forget to change it here as well)

We could as well use a deterministic hard coded storage address if this is more clear

Reply:
created a `shared_storage.cairo` file to make it very explicit. Both `library.cairo` and `uninitialized_account.cairo` will import these variables.

Reply:
see my comment, just double triple check class hash impact if anything changes in this file (name, content)